src/main/java/fi/aalto/cs/apluscourses/intellij/actions/CourseProjectAction.java

```diff
@@ -73,7 +73,6 @@
 
   private final ExecutorService executor;
 
-  // TODO: don't hard code this list
```

Comment:
This shouldn't be removed since the TODO hasn't actually been fixed :smile:

Reply:
I'd remove as the issue for this exists: https://github.com/Aalto-LeTech/intellij-plugin/issues/522 , overall, I suggest to create **ISSUEs** instead of such `TODOs`, on the other hand, the dead man's word ☠️

Reply:
Yeah it's good to have an issue, but TODO's are also easy to search for in the code and removing gives the impression that the issue has been fixed, but it hasn't :smile: So I would rather keep it (it's almost like cheating just to silence sonar)

Reply:
I'd say both TODOs and issues have their uses (but as you reasoned here, no both of them should be applied to the same problem).

TODOs can be used to keep track of minor refactoring needs (that shouldn't have a ticket of their own).

Reply:
create an issue, put it into the comments everywhere as a pointer???